pkg/controller/clusters/ipvalidator.go

```diff
@@ -41,7 +41,7 @@ func (i *IPValidator) ValidateControlPlaneIP(ctx context.Context, log logr.Logge
 	if capiCluster != nil {
 		// If CAPI cluster exists, the control plane IP has already been validated,
 		// and it's possibly already in use so no need to validate it again
-		log.V(3).Info("CAPI cluster already exists, skipping control plane IP validation")
```

Comment:
change the log level here as well?
https://github.com/aws/eks-anywhere/blob/c4a3d14e7ec7aa58d91e5ad02a550ce269109118/pkg/providers/vsphere/reconciler/reconciler.go#L221

Reply:
Sorry, what is the question? :)
I'm changing the log level of this line so the log actually shows, since this was level 3 and now we default to 0.

Reply:
I mean we should change the vsphere storage class log level to 0 as well